docs/Stonesense.rst

```diff
@@ -0,0 +1,83 @@
+:orphan:
+
+.. stonesense:
```

Comment:
@lethosor this is very embarrassing, but I made a typo here.  In order to actually make the links (and pass the build), this line must be `.. _stonesense:` - it's missing the underscore. I'm so sorry!